app/components/support_interface/reference_with_feedback_component.rb

```diff
@@ -254,6 +273,14 @@ def feedback_status_colour(reference)
       end
     end
 
+    def feature_active_and_feedback_provided?
+      FeatureFlag.active?(:show_reference_confidentiality_status) && reference.feedback_provided?
+    end
```

Comment:
```suggestion
```

The `feedback_provided?` conditional will be relevant when the feature flag is removed, so it should be moved to where it is relevant. It has nothing to do with the feature flag itself, so this will be confusing when we come to get rid of the flag later.

Reply:
Good point. I've removed this method and moved the feature flag check as well as the feedback check up to where we are inserting the row.